packages/gatsby-legacy-polyfills/src/exclude.js

```diff
@@ -0,0 +1,79 @@
+const {
+  targets: listOfPolyfillsByTarget, // object with targets for each module
+} = require(`core-js-compat`)({
+  // get polyfills for either IE11 or chrome > 63 or safari < 12.1
+  // chrome < 64 has a few bugs, the browser is over 2 years old so I feel that it's okay to not include them.
+  // seems like there are a few bugs in safari 12 & 12.1, I doubt that our users will hit it + ios has a great adoption rate.
+  targets: `IE 11, chrome > 63, safari <= 12.1`,
```

Comment:
Why chrome 63+? Wouldn't that mean all chrome users get hit by this regardless? Or is it stilll going to be selective?

Reply:
chrome < 64 has a few bugs, the browser is over 2 years old so I feel that it's okay to not include them.

We don't want to bloat the bundle with polyfills that aren't necessary because of bugs that are super edge casy.

Reply:
The part that confuses me is that this adds polyfills for old explorer, old safari, and _new_ chrome. Bloating the bundle for old browsers is fine. Bloating the bundle for new browser seems not what we want here. Please confirm?

Reply:
So the polyfill chunk will only contain polyfills necessary for IE11, safari <= 12.1. We don't want to exclude polyfills that are necessary for chrome > 63 like promise.finally.

https://github.com/gatsbyjs/gatsby/pull/25159/files#diff-a755db994786f725fb2c1beabd946fb8R12-R18

Reply:
(the keys have different functions and are not a incl/excl list for the same step)

Reply:
is it more clear now?